server.js

```diff
@@ -18,7 +18,9 @@ app.use(
     origin: [
       "http://localhost:3000",
       "http://127.0.0.1:3000",
-      "https://deploy-preview-55--100devstracker.netlify.app"
+      "https://deploy-preview-55--100devstracker.netlify.app",
```

Comment:
I believe the PR with the corresponding netlify preview has been merged. Maybe this should be updated to be the production url instead? `https://100devstracker.netlify.app/`